start-from-zero/index.html

```diff
@@ -58,11 +63,11 @@
                     </div>
                 </div>
                 
-                <div class="column-10"></div>
-                <div class="column-30">
-                        <div class="image-feedback">
-                            <img src="assets/results.jpg" alt="new feedback">
-                        </div>
+                <div class="col-md-1"></div>
```

Comment:
Does [this](https://getbootstrap.com/docs/3.3/css/#grid-offsetting) help achieve what you want, while not polluting the DOM with unneeded elements?